codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala

```diff
@@ -298,13 +298,13 @@ object ActionServiceSourceGenerator {
         |  extends ActionProvider[${service.className}] {
         |
         |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor.findServiceByName("${service.fqn.protoName}")
+        |    ${typeName(service.descriptorObject)}.javaDescriptor.findServiceByName("${service.fqn.protoName}")
         |
         |  override final def newHandler(context: ActionCreationContext): ${service.handlerName} =
         |    new ${service.handlerName}(actionFactory(context))
         |
         |  override final def additionalDescriptors: immutable.Seq[Descriptors.FileDescriptor] =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor ::
+        |    ${typeName(service.descriptorObject)}.javaDescriptor ::
```

Comment:
This change looks good, but the test trips over it. Likely because the test still uses a 'Java-style' model, I'll PR making that easier.

Reply:
(PR'ed #481, though not sure anymore it's the same problem)